village/src/components/SmurfForm.js

```diff
@@ -12,7 +13,20 @@ class SmurfForm extends Component {
 
   addSmurf = event => {
     event.preventDefault();
-    // add code to create the smurf using the api
+      axios.post('http://localhost:3333/smurfs', this.state)
+        .then(response => {
+          this.setState({
+            smurf: {
+              name: '',
+              age: '',
+              height: ''
+            }
+          })
+          window.location.reload();
```

Comment:
instead of using window.location.reload, you should pass down a method on props from App that sets state on App. Then you can call that function here and pass the new smurf up to it